src/components/Signin.jsx

```diff
@@ -0,0 +1,281 @@
+                              <label className="h5 contact-label">New to Social-Scheduler ? ,then Register here</label>
+                              <Button
+                                type="submit"
+                                className="btn btn-outline-success rounded-pill shadow"
+                                width="30%"
+                              >
+                                SignUp
+                              </Button>
+                            </div>
+                            {/* social media links for login */}
+                            <div>
+                              <Seperator >
+                                <h2><span>OR</span></h2>
+                              </Seperator>
+                              <DisplayForLinks >
+                                  <li>
+                                    <AuthLink  
+                                      color ="white" bgColor="#3B5998" 
+                                      href="#" className="btn"
+                                    >
+                                      <i className="fa fa-facebook fa-fw"></i> Signin with Facebook
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      bgColor="black" color="white" 
+                                      hoverBgColor="darkgrey" href="#" 
+                                      className=" btn"
+                                    >
+                                      <i className="fa fa-github "></i> Signin with Github
+                                    </AuthLink>
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      color="blue" bgColor="lightblue" 
+                                      href="#" className=" btn" 
+                                      contentTextAlign="center" IconSize="32px"
+                                      IconColor="blue" IconFloat="left" 
+                                      IconBgColor="white" hoverIconColor="black"  
+                                    >
+                                      <i className="fa fa-linkedin-square " /><p >Signin with Linkedin</p>
+                                    </AuthLink >
+                                  </li>
+                                  <li >
+                                    <AuthLink 
+                                      color="white" bgColor="#FE3E1B" 
+                                      border="1px solid gray" hoverBgColor="#DE3D1F" 
+                                      href="#" className=" btn" 
+                                      IconColor="white" hoverIconColor="black"
+                                    >
+                                      <i className="fa fa-google "></i> Signin with Google
```

Comment:
Kindly correct the color code for linkedIn and google Icons.

Reply:
what happened?, isnt the linkedin icon color white, and google icon white, i referred the link which you have sent for google icon.